blueprints/networking/f5-lb/variables.tf

```diff
@@ -0,0 +1,208 @@
+/**
```

Comment:
there are too many variables here, can we

- remove what is not essential and easy to change in the code (e.g. the load balancer configurations)
- group logically in `object`-typed variables (e.g. `instance_configs`, `f5_config`, etc)